### ISSUE TITLE
The performance result on GoPro_Large dataset 

### ISSUE THREAD

**Issue by pp00704831** [SeungjunNah/DeepDeblur-PyTorch#21]
Hello,

I downloaded your GoPro_Large training results and tested the PSNR.
it is able to achieve 30.4 in PSNR , but why it is about 29.08 in PSNR released in your paper?

**Comment by SeungjunNah** [comment]
Hi @pp00704831,

The accuracy difference comes from [the implementation changes](https://github.com/SeungjunNah/DeepDeblur-PyTorch#differences-from-the-original-code).
RGB range change, loss change from L2 to L1, more epochs, etc.
Those changes are more like training tricks rather than academic contributions but matters in terms of accuracy.

**Comment by pp00704831** [comment]
Hi @SeungjunNah ,
I got it, thanks you for your fast reply and devotion to deblurring!
